src/main/java/net/dv8tion/jda/core/entities/Member.java

```diff
@@ -62,15 +98,66 @@
      */
     String getNickname();
 
+    /**
+     * Retrieves the Name displayed in the official Discord Client.
+     *
+     * @return
+     *      The Nickname of this Member or the Username if no Nickname is present.
+     */
     String getEffectiveName();
 
+    /**
+     * The roles applied to this Member.
+     *
+     * @return
+     *      An immutable List of {@link net.dv8tion.jda.core.entities.Role Roles} for this Member.
+     */
     List<Role> getRoles();
 
+    /**
+     * An {@link java.awt.Color Color} this Member is displayed with.<br>
```

Comment:
This is incorrect grammar. Change `An` the `The`.
Probably should be a bit more extensive as to what this actually is.

"The {Color} of this Member's name in a Guild. This is determined by the color of the highest role assigned to them that does not have the default color. If all roles have default color, this returns null".
Atleast, I think it returns null. Need to check.